airflow/providers/apprise/provider.yaml

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+package-name: apache-airflow-providers-apprise
+
+name: Apprise
+
+description: |
+    `Apprise <https://github.com/caronc/apprise>`__
+
+suspended: false
+
+versions:
+  - 1.0.0
+
+integrations:
+  - integration-name: Apprise
+    external-doc-url: https://github.com/caronc/apprise
+    tags: [software]
+
+dependencies:
+  - apache-airflow>=2.2.0
```

Comment:
This is not right.
min version needs to be 2.4.0

Reply:
something is odd here...
There doesn't match to 
```
    "deps": [
      "apache-airflow>=2.4.0",
      "apprise"
    ],
```
    
that you have on `generated/provider_dependencies.json`

Reply:
@eladkal @potiuk the static issues are resolved and `provider.yaml` is updated as well, please take a look